README.md

```diff
@@ -17,26 +17,48 @@ Before moving into installing all the gems used, please make sure that you have
 
 After ruby is installed, install `bundler` using `gem install bundler` in your terminal. Bundler is a package (gem) manager for ruby. You can read about it at [Bundlers' website](https://bundler.io/) .
 
-We use postgres as our database. If your machine doesn't have postgres installed, please do so. You can follow [Postgres installation guide](https://wiki.postgresql.org/wiki/Detailed_installation_guides) .
+We use postgres as our database. If your machine doesn't have postgres installed, please do so. You can follow [Postgres installation guide](https://wiki.postgresql.org/wiki/Detailed_installation_guides).
 
-### Installing
+### Configure environment variables
+
+You will need a few environment variables to configure the application to call Cumulus. Create a copy of `example.env`:
+
+```bash
+cp example.env .env
+```
+
+Update `.env` with the parameters for calling the Cumulus API (CUMULUS_REST_API, EARTHDATA_USERNAME and EARTHDATA_PASSWORD).
+
+### Run the app on your development machine
 
 Once ruby and postgres is properly installed, change directory into the project folder if you have not already. `cd user-metadata-form`.
 
 Then we need to install the gems used in the project, prepare the database etc.
 
-1. `bundle install`
-2. `bundle exec rails db:create`
-3. `bundle exec rails db:migrate`
+```bash
+bundle install
+bundle exec rails db:create
+bundle exec rails db:migrate
+# Wait for the migration to complete.
+# Source environment variables used to call Cumulus.
+source .env
```

Comment:
Using `source` here won't work because the `.env` file does not `export` the vars. However, if we were to `export` the vars, then using `docker run --env-file .env ...` would complain (i.e., we cannot export them when using `.env` as a docker `env-file` because docker expects only lines of the form `name=value`, not `export name=value`, or comment lines).

However, this is unnecessary anyway, because we're using the `dotenv` gem, which automatically makes use of the `.env` file, so you can simply remove lines 43-44.